### PR TITLE
[New post] Java REST API with Helidon

### PR DESCRIPTION
**Pull request by mraible**
This blog post has the following features:

- [x] A GitHub Repository with a polished README
- [x] A GitHub Repository under the github.com/oktadev account
- [x] A title that's approved by Dev Advocacy
- [x] A URL that's approved by Dev Advocacy
- [x] The content has been run through Grammarly (https://www.grammarly.com/)
- [x] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2022-06-01-java-rest-api-helidon.adoc [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+
+toc::[]
+
+== Install a JDK with GraalVM
+
+Use SDKMAN to install Java 17 with GraalVM
+
+  sdk install java 22.1.0.r17-grl
+
+== Generate an OAuth 2.0 Access Token
```
Comment (havi) [inline _source/_posts/2022-06-01-java-rest-api-helidon.adoc]:
```suggestion
== Generate an OAuth 2.0 access token
```

File: _source/_posts/2022-06-01-java-rest-api-helidon.adoc [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+
+. Install the https://cli.okta.com/[Okta CLI] and run `okta register` to sign up for a new account. If you already have an account, run `okta login`.
+
+. Run `okta apps create spa`. Set `oidcdebugger` as an app name and press **Enter**.
+
+. Use `\https://oidcdebugger.com/debug` for the Redirect URI and set the Logout Redirect URI to `\https://oidcdebugger.com`.
+
+. Navigate to the https://oidcdebugger.com/[OpenID Connect Debugger website].
```
Comment (havi) [inline _source/_posts/2022-06-01-java-rest-api-helidon.adoc]:
```suggestion
. Navigate to the https://oidcdebugger.com/[OpenID Connect debugger website].
```

File: _source/_posts/2022-06-01-java-rest-api-helidon.adoc [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+
+. Add your Okta endpoints to `src/main/resources/META-INF/microprofile-config.properties`.
++
+[source,properties]
+----
+mp.jwt.verify.publickey.location=https://{yourOktaDomain}/oauth2/default/v1/keys
+mp.jwt.verify.issuer=https://{yourOktaDomain}/oauth2/default
+----
+
+=== Run and Test Your Helidon REST API with HTTPie
```
Comment (havi) [inline _source/_posts/2022-06-01-java-rest-api-helidon.adoc]:
```suggestion
=== Run and test your Helidon REST API with HTTPie
```

File: _source/_posts/2022-06-01-java-rest-api-helidon.adoc [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+
+. Start your Helidon app:
+
+  ./target/helidon
+
+. Test your API with an access token.
+
+  http :8080/hello Authorization:"Bearer $TOKEN"
+
+== Startup Time Comparison
```
Comment (havi) [inline _source/_posts/2022-06-01-java-rest-api-helidon.adoc]:
```suggestion
== Startup time comparison
```

File: _source/_posts/2022-06-01-java-rest-api-helidon.adoc [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+
+. Run each image three times before recording the numbers, then each command five times
```
Comment (havi) [inline _source/_posts/2022-06-01-java-rest-api-helidon.adoc]:
```suggestion
. Run each image three times before recording the numbers, then each command five times.
```

There's a "1." with no "2." to follow.

File: _source/_posts/2022-06-01-java-rest-api-helidon.adoc [hunk 6]
```diff
@@ -0,0 +1,344 @@
...
+      title: 'Java Framework'
+    }
+  };
+  var chart = new google.visualization.BarChart(document.getElementById('startup-times'));
+  chart.draw(data, options);
+}
+</script>
+++++
+
+== Memory Usage Comparison
```
Comment (havi) [inline _source/_posts/2022-06-01-java-rest-api-helidon.adoc]:
```suggestion
== Memory usage comparison
```

**Review by havi — CHANGES_REQUESTED** [review]
Minor nits re: section headings

File: _source/_posts/2022-06-02-java-rest-api-helidon.adoc [hunk 7]
```diff
@@ -0,0 +1,339 @@
+---
+layout: blog_post
+title: "Create a Java REST API with Helidon"
```
Comment (mraible) [inline _source/_posts/2022-06-02-java-rest-api-helidon.adoc:3]: "Fire up a Java REST API with Helidon" might be good too. What do you think @havi?

File: _source/_posts/2022-06-02-java-rest-api-helidon.adoc [hunk 8]
```diff
@@ -0,0 +1,339 @@
...
+import java.util.Optional;
+
+import static javax.ws.rs.core.MediaType.TEXT_PLAIN;
+
+@Path("/hello")
+public class HelloResource {
+
+    @GET
+    @Path("/")
+    @Produces(TEXT_PLAIN)
```
Comment (mraible) [inline _source/_posts/2022-06-02-java-rest-api-helidon.adoc]:
```suggestion
    @GET
    @Produces(TEXT_PLAIN)
```

File: _source/_posts/2022-06-02-java-rest-api-helidon.adoc [hunk 9]
```diff
@@ -0,0 +1,338 @@
...
+mvn -U archetype:generate -DinteractiveMode=false \
+   -DarchetypeGroupId=io.helidon.archetypes \
+   -DarchetypeArtifactId=helidon-quickstart-mp \
+   -DarchetypeVersion=2.5.0 \
+   -DgroupId=com.okta.rest \
+   -DartifactId=helidon \
+   -Dpackage=com.okta.rest
+----
++
+TIP: You can also install the https://helidon.io/docs/v2/#/about/05_cli[Helidon's CLI] and run `helidon init`.
```
Comment (havi) [inline _source/_posts/2022-06-02-java-rest-api-helidon.adoc]:
```suggestion
TIP: You can also install https://helidon.io/docs/v2/#/about/05_cli[Helidon's CLI] and run `helidon init`.
```
typo

File: _source/_posts/2022-06-02-java-rest-api-helidon.adoc [hunk 10]
```diff
@@ -0,0 +1,338 @@
...
+  var chart = new google.visualization.BarChart(document.getElementById('memory-usage'));
+  chart.draw(data, options);
+}
+</script>
+++++
+
+.What about the MacBook Pro M1 Max?
+****
+
+My MacBook Pro (16-inch, 2021) with Apple M1 Max builds _much_ faster, apps startup 2x faster, but they use more memory.
```
Comment (havi) [inline _source/_posts/2022-06-02-java-rest-api-helidon.adoc]:
```suggestion
My MacBook Pro (16-inch, 2021) with Apple M1 Max builds _much_ faster, apps start up 2x faster, but they use more memory.
```
typo

**Review by havi — CHANGES_REQUESTED** [review]
a couple of small typos.

**Pull request merged**